modules/markup_field/markup_field.info.yml

```diff
@@ -0,0 +1,7 @@
+type: module
+name: Markup field
+description: Stores the rendered output along with the dependent assets.
+package: Field types
+core: 8.x
+dependencies:
+  - drupal:hal
```

Comment:
shouldn't be there, I don't see how it is required?

Reply:
this is leftover from my former solution, it was required when this field was normalized (during replication). because default normalizers screwed serialized values so I used custom normalizer for markup field. but i'll remove it now and let's see if custom normalizer is required, would be cool if not